src/components/labelPrinting/LabelPrintingForm.vue

```diff
@@ -112,114 +95,74 @@
 
 <script>
 import SuffixList from '@/config/SuffixList'
+import {
+  createSuffixDropdownOptions,
+  createSuffixItems,
+  createLabelsFromBarcodes,
+} from '@/lib/LabelPrintingHelpers'
+import { getCurrentDate } from '@/lib/DateHelpers'
 import { mapActions } from 'vuex'
 
 const defaultForm = () => ({
-  barcode: null,
-  selectedSuffix: null,
-  selectedNumberOfLabels: null,
-  selectedPrinterName: null,
-  copies: null,
+  sourceBarcodeList: null,
+  suffix: null,
+  numberOfLabels: null,
+  printerName: null,
+  copies: '1',
```

Comment:
Does this want to be an integer, rather than a string?

Reply:
Maybe. Not sure it makes a difference. I was thinking that we could remove it all together but wasn't sure if there were any side effects.